AirCasting/ABConnector/HexMessagesBuilder.swift

```diff
@@ -13,7 +13,12 @@ struct HexMessagesBuilder {
         func getTimezoneOffsetInHours() -> Int {
             let calendar = Calendar.current
             let timeZone = calendar.timeZone
-            let hoursOffset = Int(timeZone.secondsFromGMT() / 3600)
+            var offset = timeZone.secondsFromGMT()
+            // We need to send *standard* offset, ignoring DST because AirBeam is calculating DST anyway
+            if (timeZone.isDaylightSavingTime()) {
+                offset = offset - Int(timeZone.daylightSavingTimeOffset())
+            }
+            let hoursOffset = Int(offset / 3600)
```

Comment:
I'm not sure getting timezone offset in integer hours is the correct way. Not all timezones are nicely rounded to full hour, for example Indian standard is +5:30 or Nepal with its +5:45.

Reply:
I agree the way we configure AirBeam and handle TZ is daft. However this is what AirBeam expects (hour offset as Int) and there is not so much we can do about it. I guess there are no AB users in weird timezones...yet? ;) I'll poit it out to Michael though